spec/lcm/integration/lcm_end_to_end_spec.rb

```diff
@@ -0,0 +1,255 @@
+require 'pp'
+require 'aws-sdk'
+require 'tempfile'
+require 'csv'
+
+require_relative 'support/constants'
+require_relative 'support/configuration_helper'
+require_relative 'support/lcm_helper'
+require_relative 'shared_examples_for_synchronization_bricks'
+require_relative 'shared_contexts_for_lcm_specs'
+
+# global variables to simplify passing stuff between shared contexts and examples
+$master_projects = []
+$client_projects = []
+$master = false
+
+describe 'the whole life-cycle' do
+  include_context 'lcm bricks'
+
+  describe '1 - Initial Release' do
```

Comment:
Similar code found in lcm_end_to_end_spec.rb:20, lcm_end_to_end_spec.rb:140 (mass = 124)